src/modules/enrollment/components/dashboardPages/EnrollmentServicesPage.tsx

```diff
@@ -0,0 +1,109 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Button, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import { useState } from 'react';
+
+import { ColumnDef } from '@/components/elements/table/types';
+import TitleCard from '@/components/elements/TitleCard';
+import { useEnrollmentDashboardContext } from '@/components/pages/EnrollmentDashboard';
+import NotFound from '@/components/pages/NotFound';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import { parseAndFormatDate, serviceDetails } from '@/modules/hmis/hmisUtil';
+import { useServiceDialog } from '@/modules/services/hooks/useServiceDialog';
+import {
+  GetEnrollmentServicesDocument,
+  GetEnrollmentServicesQuery,
+  GetEnrollmentServicesQueryVariables,
+  ServiceFieldsFragment,
+} from '@/types/gqlTypes';
+
+const baseColumns: ColumnDef<ServiceFieldsFragment>[] = [
+  {
+    header: 'Date Provided',
+    linkTreatment: true,
+    render: (e) => parseAndFormatDate(e.dateProvided),
+  },
+  {
+    header: 'Category',
+    render: 'serviceType.category' as keyof ServiceFieldsFragment,
+  },
+  {
+    header: 'Service Type',
+    render: 'serviceType.name' as keyof ServiceFieldsFragment,
+  },
+  {
+    header: 'Details',
+    render: (e) => (
+      <Stack>
+        {serviceDetails(e).map((s, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <Typography key={i} variant='body2'>
```

Comment:
can the array item be the key? Or are the items non-unique?
```suggestion
          <Typography key={s} variant='body2'>
```

Reply:
They're not necessarily unique